pynest/examples/eprop_plasticity/eprop_supervised_classification_evidence-accumulation.py

```diff
@@ -189,28 +187,42 @@
 n_rec = n_ad + n_reg  # number of recurrent neurons
 n_out = 2  # number of readout neurons
 
-
-params_nrn_reg = {
+params_nrn_out = {
     "C_m": 1.0,  # pF, membrane capacitance - takes effect only if neurons get current input (here not the case)
-    "c_reg": 2.0,  # firing rate regularization scaling - double the TF c_reg for technical reasons
     "E_L": 0.0,  # mV, leak / resting membrane potential
-    "f_target": 10.0,  # spikes/s, target firing rate for firing rate regularization
-    "gamma": 0.3,  # scaling of the pseudo derivative
+    "eprop_isi_trace_cutoff": 100,  # cutoff of integration of eprop trace between spikes
     "I_e": 0.0,  # pA, external current input
-    "regular_spike_arrival": True,  # If True, input spikes arrive at end of time step, if False at beginning
-    "surrogate_gradient_function": "piecewise_linear",  # surrogate gradient / pseudo-derivative function
-    "t_ref": 5.0,  # ms, duration of refractory period
+    "regular_spike_arrival": False,  # If True, input spikes arrive at end of time step, if False at beginning
```

Comment:
This is in principle possible. We are currently working on a PR https://github.com/JesusEV/nest-simulator/pull/20 to remove `regular_spike_arrival` entirely. On further reflection, the evidence accumulation task does not function without batch learning. However, batch learning is not implemented for the non-bsshslm_2020 model. Therefore, it might be best to remove the non-bsshslm_2020 version of this task altogether.